usaspending_api/transactions/delta_models/summary_state_view.py

```diff
@@ -0,0 +1,55 @@
+SUMMARY_STATE_VIEW_COLUMNS = {
+    "duh": "STRING",
+    "action_date": "STRING",
+    "fiscal_year": "STRING",
+    "type": "STRING",
+    "distinct_awards": "STRING",
+    "pop_country_code": "STRING",
+    "pop_state_code": "STRING",
+    "counts": "STRING",
+    "transaction_id": "STRING",
+    "compound_geo_pop": "STRING",
+    "ordered_action_date": "STRING",
+}
+
+summary_state_view_string = rf"""
+ CREATE OR REPLACE TABLE {{DESTINATION_TABLE}} (
```

Comment:
I believe this need to be split out into two different sql strings similar to transaction_search and award_search: one for creating the empty table and another for inserting the data into the table.

Reply:
Just split it.